web/src/pages/VendorSignUp.js

```diff
@@ -74,8 +74,8 @@ const VendorSignUp = ({hideHeader, showHeader, setLoginState}) => {
 			'name': name, 
 			'email': email, 
 			'password': password,
-			'lng': selectedCoord.lng(), 
-			'lat': selectedCoord.lat(), 
+			'lng': selectedCoord.lng, 
+			'lat': selectedCoord.lat, 
```

Comment:
Using selectedCoord.lng didn't work when I tried, did you test and make sure that latitude and longitude is set correctly in the database?

Reply:
The call to the lng/lat functions now happen in GoogleMap component (see the changes in that file). I changed it, because I needed map.panTo to use lat()/lng() values as its arguments & since we need to make these calls in the map component anyway, the info we give outside can just be these values (and so the functions don't need to be called again).

In other words, vendor signup still uses the same values; nothing really changed. It just doesn't receive functions & perform the calls itself. It receives the values in the first place.

I tested it, ofc. Both VendorSignup and VendorProfile work correctly. This is the test I did before I pushed the commit (can be recreated, if you wish):

- I went to the vendor signup page & filled in details for a new vendor.
- Changed the map location a few times, just to make sure nothing unexpected happens. Completed the sign-up.
- Went to the vendor's profile page. Saw that the map is showing the correct location.
- Changed & updated vendor's location. Saw that the map on the profile page changed to the correct location.
- Received the dopamine rush of doing something right ^^